crm_claim_rma/models/claim_line.py

```diff
@@ -111,30 +112,29 @@ class ClaimLine(models.Model):
     warning = fields.Selection(WARRANT_COMMENT,
                                'Warranty', readonly=True,
                                help="If warranty has expired")
-    display_name = fields.Char('Name', compute='_get_display_name')
+    display_name = fields.Char('Name', compute='_compute_display_name')
 
     @api.model
     def get_warranty_return_partner(self):
-        return self.env['product.supplierinfo']._columns[
-            'warranty_return_partner'
-        ].selection
+        return self.env['product.supplierinfo'].warranty_return_partner
```

Comment:
Change this to: `self.env['product.supplierinfo'].fields_get('warranty_return_partner')['warranty_return_partner']['selection']` and you are golden. Will test the rest tomorrow.